css/sass/spacing-classes.scss

```diff
@@ -0,0 +1,100 @@
+/**
+ * @margin-and-padding.scss
+ * global classes for margin and padding
+ * naming order follows the property shorthand:
+ * • top • right • bottom • left
+ * m or p (margin or padding)
+ * side  (top • right • bottom • left)
+ * amount and unit (px • em • rem)
+ * example generated classes:
+ *   .m-trbl--1px, .m-t--1em, m-b--1rem
```

Comment:
I disagree with that specific implementation but I guess it's fine to leave it in. It's up to whoever uses it to strip out what they don't need. 

If you could just fix the link in the README.md we'll get this merged!